simpleprophet/setup.py

```diff
@@ -0,0 +1,59 @@
+"""Mozilla simpleprophet forecast framework."""
+
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='simpleprophet',
+    use_scm_version={"root": "..", "relative_to": __file__},
+    setup_requires=['setuptools_scm'],
+    description=__doc__,
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://github.com/mozilla/forecasting',
+    author='Mozilla',
+
+    # Classifiers help users find your project by categorizing it.
+    #
+    # For a list of valid classifiers, see https://pypi.org/classifiers/
+    classifiers=[  # Optional
+        # How mature is this project? Common values are
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 3 - Alpha',
+
+        # Indicate who your project is intended for
+        'Intended Audience :: Developers',
+
+        # Pick your license as you wish
+        'License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)',
+
+        # Specify the Python versions you support here. In particular, ensure
+        # that you indicate whether you support Python 2, Python 3 or both.
+        # These classifiers are *not* checked by 'pip install'. See instead
+        # 'python_requires' below.
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+    ],
+
+    install_requires=['fbprophet>=0.5'],
```

Comment:
Does this section also need to reference the dependencies specified in requirements.txt?

Reply:
Yes, will do.